### PR TITLE
Fix/bookmark widget host

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[維運] 文章頁收藏相關問題](https://app.asana.com/0/1203699264568808/1206379506612355/f)

# Notice

# Dependency

### CODE REVIEW

File: src/data-loaders/bookmark-widget.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ const _ = {
 const reduxStatePropKey = twreporterRedux.reduxStateFields
 
 const host = {
-  master: 'http://testtest.twreporter.org:3000',
-  preview: 'http://testtest.twreporter.org:3000',
```
Comment (Aylie-Chou) [inline src/data-loaders/bookmark-widget.js:17]: `preview` branch is for keystone preview
I think we could keep it since it's still under maintenance

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
as comment~

File: src/data-loaders/bookmark-widget.js (hunk 1 above)
Comment (liruchen32) [reply src/data-loaders/bookmark-widget.js:17]: fixed

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**